### ISSUE TITLE
Please remove the "Irish Car Bomb" drink

### ISSUE THREAD

**Issue by ItsMeThom** [space-wizards/space-station-14#34102]
## Description

This drink is woefully hateful in naming and trivialises a pretty serious time of Irish history, and that doesnt seem right to include in a game that promotes inclusivity and anti-hate. This is the equivalent of naming a drink a 9/11.

**Comment by lzk228** [comment]
yeah, makes sense, but maybe we can just rename it?
![image](https://github.com/user-attachments/assets/81a2e6e5-cb40-49ea-9eb5-a644cbf03860)

**Comment by Errant-4** [comment]
Yeah this is reasonable, change it

**Comment by IamVelcroboy** [comment]
I don't care what you call it, they're fucking delicious!

For real though, if you're that worried about it, just remove "Irish" and call it a "Car Bomb."

**Comment by Alpaccalypse** [comment]
Made a PR to address this issue:
https://github.com/space-wizards/space-station-14/pull/34107

Is there a built-in way of linking PRs to issues that I am missing? (Or does this do it? - #34107 )

**Comment by ScarKy0** [comment]
> Is there a built-in way of linking PRs to issues that I am missing? 

Iirc adding "Fixes #XYZ" to the pr description

**Comment by Sarahon** [comment]
> ## Description
> This drink is woefully hateful in naming and trivialises a pretty serious time of Irish history, and that doesnt seem right to include in a game that promotes inclusivity and anti-hate. This is the equivalent of naming a drink a 9/11.

This really isn't the equivalent to naming a drink "9/11", that is such an insane comparison. I really don't get how this is acceptable at all lmfao

**Comment by PJB3005** [comment]
> > ## Description
> > This drink is woefully hateful in naming and trivialises a pretty serious time of Irish history, and that doesnt seem right to include in a game that promotes inclusivity and anti-hate. This is the equivalent of naming a drink a 9/11.
> 
> This really isn't the equivalent to naming a drink "9/11", that is such an insane comparison. I really don't get how this is acceptable at all lmfao

[Wikipedia disagrees](https://en.wikipedia.org/wiki/Irish_car_bomb_(cocktail)). Please be respectful.

**Comment by Sarahon** [comment]
> > > ## Description
> > > This drink is woefully hateful in naming and trivialises a pretty serious time of Irish history, and that doesnt seem right to include in a game that promotes inclusivity and anti-hate. This is the equivalent of naming a drink a 9/11.
> > 
> > 
> > This really isn't the equivalent to naming a drink "9/11", that is such an insane comparison. I really don't get how this is acceptable at all lmfao
> 
> [Wikipedia disagrees](https://en.wikipedia.org/wiki/Irish_car_bomb_(cocktail)). Please be respectful.

I've no idea what you mean right now PJB. I'm talking about the comparison - the troubles and 9/11 are very different situations to compare, it's disrespectful to compare them both. And I am in/lived in NI my whole life(Yes my parents lived through the troubles, which this drink refers to) and that is just a crazy comparison to me.

**Comment by Sarahon** [comment]
> I don't think people are comparing 9/11 to The Troubles in terms of scale or as a gotcha, rather as a hypothetical on naming something after potentially sensitive history. YMMV depending on personnel beliefs I guess.

Yeah I dunno, it's kinda just one of those things that exist. It's not meant to be insensitive or offensive, it just is what it's called. It's just kinda blowing my mind that this has been made an issue when it's legit just a drink anyone could order at any moment, in the place the carbombings themselves started. Though, we don't really scare from our history, the Irish way I guess.

**Comment by ninruB** [comment]
> Wiki isn't a great source of info to be real.

Kind of a broad generalization, but it's true in this case. The Wikipedia page has 3 sources listed, all 3 are heavily opinionated news articles providing no actual data or solid ground.

**Comment by Errant-4** [comment]
For the record, there is no need for you to close issues as completed, that should happen when their linked PR is merged. Only close issues if their fix was already merged but they were not linked, so did not auto-close
